src/cox.jl

```diff
@@ -193,11 +211,11 @@ Given a matrix `M` of predictors and a corresponding vector of events, compute t
 Cox proportional hazard model estimate of coefficients. Returns a `CoxModel`
 object.
 """
-function StatsBase.fit(::Type{CoxModel}, M::AbstractMatrix, y::AbstractVector; kwargs...)
+function StatsBase.fit(::Type{CoxModel}, M::AbstractMatrix, y::AbstractVector; tol=1e-4, l2_cost=0)
     index_perm = sortperm(y)
     X = M[index_perm,:]
     s = y[index_perm]
-    _coxph(X, s; kwargs...)
+    _coxph(X, s; tol, l2_cost)
```

Comment:
```suggestion
    _coxph(X, s; tol=tol, l2_cost=l2_cost)
```
I guess here too for 1.0 support.